cloudify_cli/commands/snapshots.py

```diff
@@ -0,0 +1,87 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+"""
+Handles all commands that start with 'cfy snapshots'
+"""
+
+from cloudify_cli import utils
+from cloudify_cli.logger import get_logger
+from cloudify_cli.utils import print_table
+
+
+def restore(snapshot_id, without_deployments_envs):
```

Comment:
Additionally, sometimes you might just want to "tidy up" your ES data when it gets corrupt, without touching dep envs (it lasts long and apart from that you must delete those envs before).